src/resources/pandoc/datadir/_format.lua

```diff
@@ -279,7 +295,7 @@ local function _main()
     isRawLatex = isRawLatex,
     is_raw_latex = isRawLatex,
     isFormat = isFormat,
-    is_format = is_format,
+    is_format = isFormat,
```

Comment:
Wow, nice catch. I wish there was a way for our Lua LSP to report these to us...

Reply:
yes. It happens it was also unused because we provide format in API at 
`quarto.doc.is_format`
https://github.com/quarto-dev/quarto-cli/blob/19f27288b592f067364dbf0af28b605bf5fe8604/src/resources/pandoc/datadir/init.lua#L2082

and we document that one https://quarto.org/docs/extensions/lua-api.html#format-detection

We don't document `quarto.format` that we export too
https://github.com/quarto-dev/quarto-cli/blob/19f27288b592f067364dbf0af28b605bf5fe8604/src/resources/pandoc/datadir/init.lua#L1952-L1953

Reply:
We should document `quarto.format` (and anything else in the `quarto.*` namespace).

Reply:
💯 https://github.com/quarto-dev/quarto-cli/issues/4499